supercloud.md

```diff
@@ -0,0 +1,78 @@
+# Running Experiments on Supercloud
+
+## First Time Setup
+
+### Getting Access to Supercloud
+
+* If you do not yet have a supercloud account, you will need to [request one](https://supercloud.mit.edu/requesting-account). 
+* After receiving an account, follow the instructions to ssh into supercloud.
+* Once logged in to supercloud, [create an ssh key and add it to your github.com account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
+* The most important rule about using supercloud is that you should not run any intensive jobs on a login node. Instead, launch batch jobs, or use an interactive session. Both options are illustrated below.
+
+### Installing this Repository
+
+Run the following commands to install this repository.
+
+```
+# Set `PYTHONHASHSEED` to 0 by default (see our README.md for explanation).
+echo "export PYTHONHASHSEED=0" >> ~/.bashrc
+# Clone the repository.
+git clone git@github.com:Learning-and-Intelligent-Systems/predicators.git
+# Set up conda with Python 3.9.
+module unload anaconda
+module load anaconda/2021b
+conda create --name predicators python=3.9
+conda init bash  # will need to restart shell after this
+conda activate predicators
+# Install the predicators dependencies.
+cd predicators
+mkdir /state/partition1/user/$USER
+export TMPDIR=/state/partition1/user/$USER
+pip install -r requirements.txt
+# Add a shortcut for activating the conda env and switching to this repository.
+echo -e "predicate() {\n    cd ~/predicators\n    conda activate predicators\n}" >> ~/.bashrc
+```
+To test if it worked:
```

Comment:
clarify that you need to start a new shell (so that `predicate` works)

Reply:
i'll add `source ~/.bashrc` to the end of the instructions